lib/parseSignatureHeader.test.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const tap = require('tap')
+const parseSignatureHeader = require('./parseSignatureHeader')
+
+tap.test('Signature header is extracted as a dictionary.', async ({ same }) => {
+  const tests = [
+    {
+      description: 'Dictionary created from valid signature header',
+      input: { headers: { signature: 'signature=1d3g5h7,biff=buzz' } },
```

Comment:
[nit] Is the delimiter here a `,` or is there whitespace? `, `? We might need to trim any excess whitespace.